internal/output/render.go

```diff
@@ -45,10 +49,15 @@ func Render(writer io.Writer, outputFormat string, commandOutputs ...Output) (er
 		return err
 	}
 
-	// Count failed outputs
+	// Render streaming outputs and count failed ones
 	failedCount := 0
 	for _, commandOutput := range commandOutputs {
-		if _, ok := commandOutput.(Error); ok {
+		if rawOutput, ok := commandOutput.(Raw); ok {
+			defer rawOutput.Close()
+			if _, err := io.Copy(writer, rawOutput); err != nil {
+				return err
+			}
```

Comment:
I'm not too happy with this implementation, but at least it is a rather non-intrusive one.